AWSAuthSDK/Sources/AWSMobileClient/Internal/JSONHelper.swift

```diff
@@ -54,4 +54,30 @@ struct JSONHelper {
         }
         return nil
     }
+    
```

Comment:
Better to add some unit test around these methods.

Reply:
Added